mesheryctl/pkg/utils/helpers.go

```diff
@@ -201,7 +210,7 @@ func startdockerdaemon(subcommand string) error {
 			return errors.Wrapf(err, "please start Docker then run the command `mesheryctl system %s`", subcommand)
 		}
 		// wait for few seconds for docker to start
-		err = exec.Command("sleep", "20").Run()
+		err = exec.Command("sleep", "30").Run()
```

Comment:
I am always concerned with sleep commands in code. However, this might be a necessity. Can we do something like `$ systemctl status docker` and sort of run it in a goroutine and get the value.

Reply:
> I am always concerned with sleep commands in code. However, this might be a necessity. Can we do something like `$ systemctl status docker` and sort of run it in a goroutine and get the value.

I need to investigate this further, moreover "systemctl" is not present on MacOS. We discussed about the sleep time in the initial PR too. Agreed, this need to change.